change/@fluentui-react-toolbar-45625abc-30ac-4962-98fc-06370a993261.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
please review these change types again, they should not be `none` because you are changing the behaviour of the component